res/values/setup.xml

```diff
@@ -53,6 +53,7 @@
     <bool name="imprint_enabled">false</bool> 
     <bool name="recommend_enabled">true</bool>
     <bool name="feedback_enabled">true</bool>
+    <bool name="logger_enabled">true</bool>
```

Comment:
For the same reason the other are there ;) --> with this flag you can decide if the log part will be rendered in the settings screen or not. We can take it out if you want to. I just thought like with imprint, feedback etc. you would like to be able to have to feature toggle for logging also since I discussed the move to the settings screen with @jancborchardt and we agreed that after moving it to the settings screen it will also be visible in the official release.